src/components/ebay-education-notice/component.ts

```diff
@@ -0,0 +1,36 @@
+import type { Input as NoticeBaseInput } from "../components/ebay-notice-base/component";
+import type { WithNormalizedProps } from "../../global";
+
+interface SectionNoticeInput
+    extends Omit<
+        NoticeBaseInput,
+        "role" | "prefixClass" | "mainRoot" | `on${string}`
+    > {
+    dismissed?: boolean;
```

Comment:
`title` should probably be made required here.

Reply:
Title comes from notice base input. We use it from there.

Reply:
In `notice-base` it's optional, right? In `education-notice` it's required.

Reply:
Figma is showing title as optional:

<img width="838" alt="Screenshot 2024-03-14 at 1 01 36 PM" src="https://github.com/eBay/ebayui-core/assets/38065/391eadf6-c4c7-4278-ada9-f2777bb0b5c5">

Reply:
That's not what I see. Maybe another change that was made recently after I was done with the work?

<img width="965" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/4122df5b-7100-4c57-80e5-2623f8d86465">

Reply:
I'm not sure where/how you got to that Figma file, but the one linked in the story still has the title as required. What you're seeing might be older.

Reply:
Sounds good. That takes care of the actual PR concern, but I'd still like to know where that Figma screenshot came from. Having an obsolete Figma file out there is a bit concerning. @ianmcburnie ?